group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/UniformHeterogeneousAssignmentBuilder.java

```diff
@@ -75,798 +86,841 @@ public class UniformHeterogeneousAssignmentBuilder {
     private final Set<Uuid> subscribedTopicIds;
 
     /**
-     * List of subscribed members for each topic.
+     * The list of members in the consumer group, sorted by member id.
```

Comment:
nit: Should we remove the "sorted by member id"?

Reply:
Good spot, I've updated the javadoc.